pkg/volume/csi/csi_attacher.go

```diff
@@ -66,6 +66,11 @@ var _ volume.Detacher = &csiAttacher{}
 var _ volume.DeviceMounter = &csiAttacher{}
 
 func (c *csiAttacher) Attach(spec *volume.Spec, nodeName types.NodeName) (string, error) {
+	_, ok := c.plugin.host.(volume.KubeletVolumeHost)
```

Comment:
Do you know which line was segfaulting? The line number in the stack trace doesn't appear to match this file.